{
  "comments": [
    {
      "key": {
        "uuid": "fafc7978_a7d99885",
        "filename": "cluster/src/main/scala/org/midonet/cluster/data/storage/Storage.scala",
        "patchSetId": 1
      },
      "lineNbr": 89,
      "author": {
        "id": 1002751
      },
      "writtenOn": "2014-11-18T09:13:29Z",
      "side": 1,
      "message": "Minor comment: why not adding this class to ZoomException.scala?",
      "revId": "9af893dd9bc6f6a0c23e0d162a47b2d57b290827",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "fafc7978_c74adc26",
        "filename": "cluster/src/main/scala/org/midonet/cluster/data/storage/Storage.scala",
        "patchSetId": 1
      },
      "lineNbr": 89,
      "author": {
        "id": 1002760
      },
      "writtenOn": "2014-11-18T09:19:12Z",
      "side": 1,
      "message": "StorageException is a general Exception class and not particularly about ZOOM, and is actually part of the public Storage API. So I don\u0027t think it belongs there.",
      "parentUuid": "fafc7978_a7d99885",
      "revId": "9af893dd9bc6f6a0c23e0d162a47b2d57b290827",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    }
  ]
}